### PR TITLE
TG-117: Landing Page

### PR DESCRIPTION
**Pull request by nicolascostanza**
Here we developed the landing page according to the figma layout.
It has no functionalities, is purely aesthetic

### CODE REVIEW

File: src/Components/Shared/Landing/landing.module.css [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+  margin-top: 3px;
+  margin-bottom: 8px;
+  border-radius: 2px;
+  resize: none;
+}
+#descriptionImage{
+  min-width: 150px;
+  max-height: 150px;
+};
+/* .contact-us form {
```
Comment (mpueblas) [inline src/Components/Shared/Landing/landing.module.css:268]: esto esta comentado

File: src/Components/Shared/Landing/landing.module.css [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+  display: flex;
+  flex-direction: column;
+}
+.imgTablet {
+  display: none;
+}
+
+@media only screen and (max-width: 780px) {
+  main{
+    /* width: 95%; */
```
Comment (mpueblas) [inline src/Components/Shared/Landing/landing.module.css:292]: y aca

**Review by mpueblas — APPROVED** [review]
hay dos partes comentadas que habria que borrar, pero el resto lo encontré muy bien! va el approve derecho

**Review by martinalbrnz — APPROVED** [review]
Good job!

**Comment by RominaRIP** [comment]
## Testing Results

Nicolás and Federico, please check the following bug report. For more details, go to the ticket 'TG-137' in Trello: [TG-137](https://trello.com/c/5MZD2RnX/137-bug-landing-page-bug-report)

Tested locally on https://alfonso-trackgenix-app-git-tg-117-basp-m2022.vercel.app/
Test Results: Failed ⚠️

Bugs found:
- (1) Accessibility test with contrast checker failed for 'Header - Nav bar menu [hover]' and 'Footer - Social media i.fa-brands [hover]'.
- (2) Button to display the side-bar menu is an arrow
- (3) Buttons of 'Learn more' in the intro section and 'Send message' in the form section don't have a hover.
- (4) User can observe a white margin in both sides of the header page
- (5) The text-area placeholder font is not the same as the rest of the form placeholders
- (6) 'h2' tag in all sections doesn't correspond with the figma design
- (7) List elements in the 'landing_listFooter__a2ffE' section are not aligned (check image I). Also, in tablet and mobile resolution, they are not centered
- (8) Social-media HyperLinks are repeated in three different places

**Comment by RominaRIP** [comment]
## 2nd Testing Results

Nicolás, Fer and Federico, please check the following bug report. For more details, go to the ticket 'TG-137' in Trello: [TG-137](https://trello.com/c/5MZD2RnX/137-bug-landing-page-bug-report)

Tested locally on https://alfonso-trackgenix-app-git-tg-117-basp-m2022.vercel.app/
Test Results: Failed ⚠️

Bugs found:

- (1) Button to display the side-bar menu is not well aligned:

B2-nav-bugs


- (2) The text-area placeholder font is still not the same as the rest of the form placeholders. Also, the color of the 'Send message' button doesn't correspond with the color palette of the project. Maybe you can use the same color as the icons in the 'Functionalities' section: #F13312

A2-form-bugs


- (3) The following section is still not aligned in the mobile and tablet resolutions:

C2-align-bug


- (4) The social media icons are still repeated. If you want to display them in the nav-bar, we suggest to remove them from the footer. 

D2-social-media-bug


- (5) The hover in the buttons and anchors are not unified. We suggest to choose one color for the hover in all of them (social media icons, 'Send message button in form section and HyperLinks for 'Admins', 'Super admins', etc.)

- (6) We suggest to put the HyperLinks to 'Admins', 'Super admins', etc. in the fixed nav-bar, for more navigation comfort through the page:

E2-nav-bar-links

**Comment by RominaRIP** [comment]
## 3rd Testing Results

Tested locally on https://alfonso-trackgenix-app-git-tg-117-basp-m2022.vercel.app/
Test Results: Passed ✅

Some strictly esthetic errors will be reviewed in next instances.

**Review by FefeTroanes — APPROVED** [review]
😎👉🏼

**Review by ilcosme — APPROVED** [review]

**Pull request merged**